src/components/GraphFilter/GraphRefresh.tsx

```diff
@@ -79,8 +86,8 @@ export class GraphRefresh extends React.PureComponent<GraphRefreshProps> {
           disabled={this.props.disabled}
           handleSelect={this.props.setDuration}
           value={this.props.duration}
-          label={String(GraphRefresh.DURATION_LIST[this.props.duration])}
-          options={GraphRefresh.DURATION_LIST}
+          label={String(validDurations[this.props.duration])}
```

Comment:
Please see my comment of the reducer/ServerConfigState.ts file. I think this is what may become `undefined`.